testing-guidance/workflow-tests/ragdoll/ragdoll-authoring-improvements.md

```diff
@@ -0,0 +1,47 @@
+| **Joint Limit Optimization** | <ol><li>Remove the added Motion Set, by going to the AnimGraph layout → Motion Sets pane → Motion Set Management, right-clicking the MotionMatching.motionset, and selecting "Remove selected" from the context menu.</li><li>Go back to the Physics layout, select a bone with joint limit enabled in the Skeleton Outliner, and press the "Automatic joint limit setup" button at the bottom of the clusters in the Atom Render Window. A message should popup in the bottom right corner informing the user, that a Motion Set must be loaded.</li><li>Add the Motion Set back, by changing the layout to AnimGraph (from Physics), and add MotionMatching.motionset under the Motion Sets → Motion Set Management by clicking the "Load motion set from file" button".</li><li>Head back to the Physics layout, and press the "Automatic joint limit setup" button at the bottom of the clusters in the Atom Render Window. Note the automatic changes that occur on the joint limit properties in the Ragdoll pane. If no changes are seen, make some modifications on all parameters, and then click the button again. Joint limits should roughly match the movement given bone could be doing (for example the knee joint should have a cone close to 180 degrees in one direction, and narrow range of movement in the other direction).</li></ol> | <ul><li>Motion Sets can be removed, and added in the Animation Editor.</li><li>A message pops up if there is no Motion Set loaded while the user tries to use the "Automatic joint limit setup".</li><li>"Automatic joint limit setup" properly overwrites any changes made to the joint limit, and roughly matches the expected range of movement.</li></ul>  |
```

Comment:
Step 2: Move the "A message" validation to the Things to Watch Out For
Step 4: There's a lot of mixing between actions and Things to Watch Out For here.  Abstract the clear step(s) out here and move the rest into the Things to Watch Out For.

Reply:
Adjusted steps 2, and 4 as requested.